.gitignore

```diff
@@ -0,0 +1,10 @@
+# git ignore file for the Docker-Provider project
+
+/intermediate/
+/target/
+build/config.mak
+
+# Unit test files
+
+/test/code/providers/TestScriptPath.h
```

Comment:
This file is created by the build process. I didn't get into details with Yuyang about it, I didn't much care. It's created during the build process, so I put it in .gitignore.